### DISCUSSION TITLE
Import Kindle Highlights

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#753] [Feature Requests]
I use my kindle quite a bit, and while I manually import the highlights. It would be great if GitJournal could offer a service like readwise where I can just forward an email and it gets added to my git repo.

This will require significant backend work, but it seems easily doable.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by victorferreira** [comment]
That'd be awesome!

**Comment by vHanda** [comment]
This could also be implemented as a Browser Plugin by building on top of https://readwise.io/bookcision
